src/lib/DepositRecordSerializer.test.js

```diff
@@ -324,6 +319,7 @@ describe('DepositRecordSerializer', () => {
         metadata: {
           contributors: [
             {
+              affiliations: [],
```

Comment:
It is a bit unclear what's expected with the "default" (de)serilization. In a test above, all fields are given the default value if not present, e.g. for languages. Therefore, I am assuming the same happens here for the inner fields of a creatibutor. We should remove these when sending to the API (I think we are doing it already tho).